tests/testdata/examples/aws-rds-example/index.ts

```diff
@@ -0,0 +1,77 @@
+import * as vpcmod from '@pulumi/vpcmod';
```

Comment:
Can we have this under top-level ./examples? Since it is indeed a user-facing example presumably, and not just something synthetic for the implementation.

Reply:
Good point